### PR TITLE
 Updating security policy

### PR DESCRIPTION
**Pull request by shanmydell**
Updating security policy

### CODE REVIEW

File: content/docs/securitypolicy/_index.md [hunk 1]
```diff
@@ -6,9 +6,20 @@ Description: >
   Dell Container Storage Modules (CSM) Security Policy
 ---
 
+# Security Policy
 
-The CSM services/repositories are inspected for security vulnerabilities via [gosec](https://github.com/securego/gosec), 
-Instructions for reporting a vulnerability can be found on the [Dell Vulnerability Policy](https://www.dell.com/support/contents/en-in/article/product-support/self-support-knowledgebase/security-antivirus/alerts-vulnerabilities/dell-vulnerability-response-policy#:~:text=To%20report%20a%20security%20vulnerability%20or%20issue%20in%20Dell.com,instructions%20to%20reproduce%20the%20issue)
+The CSM services/repositories are inspected for security vulnerabilities via [gosec](https://github.com/securego/gosec).
```
Comment (gallacher) [inline content/docs/securitypolicy/_index.md:11]: Remove everything above "Reporting Security Issues/Vulnerabilities"

File: content/docs/securitypolicy/_index.md [hunk 2]
```diff
@@ -6,9 +6,20 @@ Description: >
...
 
+Every issue detected by `gosec` is mapped to a [CWE (Common Weakness Enumeration)](http://cwe.mitre.org/data/index.html) which describes in more generic terms the vulnerability.  The exact mapping can be found at https://github.com/securego/gosec in the `issue.go` file. The list of rules checked by `gosec` can be found [here](https://github.com/securego/gosec#available-rules).
 
-CSM recommends to stay on the [lastest release](https://github.com/dell/csm/releases/latest) of Dell Container Storage Modules to take advantage of new features, enhancements, bug fixes, and security fixes.
+In addition to this, various security checks get executed against a branch when a pull request is created/updated.  Please refer to [pull request](/docs/CONTRIBUTING.md#pull-requests) for more information.
+
+# Reporting Security Issues/Vulnerability
```
Comment (gallacher) [inline content/docs/securitypolicy/_index.md:17]: Typo - Vulnerability should be Vulnerabilities

File: content/docs/securitypolicy/_index.md (hunk 1 above)
Comment (shanmydell) [reply content/docs/securitypolicy/_index.md]: done

File: content/docs/securitypolicy/_index.md [hunk 3]
```diff
@@ -6,9 +6,20 @@ Description: >
...
+
+The Dell Container Storage Modules team and community take security bugs seriously. We sincerely appreciate all your efforts and responsibility to disclose your findings.
+
+To report a security issue, please submit the security advisory form ["Report a Vulnerability"](https://github.com/dell/csm/security/advisories/new).
+
+We will send a response along with the next steps after analyzing the report.
```
[the remark this answers is not in the document]
Comment (shanmydell) [reply content/docs/securitypolicy/_index.md]: done

File: content/docs/securitypolicy/_index.md (hunk 2 above)
Comment (shanmydell) [reply content/docs/securitypolicy/_index.md]: done

**Comment by shanmydell** [comment]
@gallacher  : Could you please review the updated file? thanks

**Review by gallacher — APPROVED** [review]

**Review by boyamurthy — APPROVED** [review]

**Pull request merged**